### ISSUE TITLE
fitHorizontally with rotated viewport

### ISSUE THREAD

**Issue by adamwimmj** [openseadragon/openseadragon#1744]
Hi, I'm having an issue where if I rotate an image 90 degrees - viewport.fitHorizontally() incorrectly zooms the image so it does not fill the width of the viewport. It works perfectly with an unrotated viewport. Am I misinterpreting how fitHorizontally works? Is this expected?

**Comment by iangilman** [comment]
Looks like it's a bug... we don't appear to be paying attention to the rotation at all in `fitHorizontally`. Just to confirm: you're rotating the viewport (`Viewport.setRotation`), not the image (`TiledImage.setRotation`), right?

**Comment by adamwimmj** [comment]
Yes I am rotating the viewport. But applying rotation to the TileImage does actually fix my issue, thanks for the suggestion.

**Comment by iangilman** [comment]
Nice! Good to know. 

Well, we'll leave this bug report for whoever might want to fix `fitHorizontally` to pay attention to the rotation.
